hack/build-marker-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ -z "$ARCH" ] || [ -z "$PLATFORMS" ] || [ -z "$MARKER_IMAGE_TAGGED_1" ] || [ -z "$MARKER_IMAGE_TAGGED_2" ]; then
+    echo "Error: ARCH, PLATFORMS, MARKER_IMAGE_TAGGED_1, and MARKER_IMAGE_TAGGED_2 must be set."
+    exit 1
+fi
+
+IFS=',' read -r -a PLATFORM_LIST <<< "$PLATFORMS"
+
+BUILD_ARGS="--no-cache --build-arg BUILD_ARCH=$ARCH -f build/Dockerfile -t $MARKER_IMAGE_TAGGED_1 -t $MARKER_IMAGE_TAGGED_2 . --push"
```

Comment:
why `--no-cache` ?
we do need it at least locally usually

Reply:
Removed in latest commit.